docs/source/cql.rst

```diff
@@ -3,10 +3,15 @@
 CQL support
 ===========
 
+Providers
+---------
+
+As of now the available providers supported for CQL filtering are limited to :ref:`Elasticsearch <Elasticsearch>` & :ref:`PostgreSQL <PostgreSQL>`.
```

Comment:
`s/&/and/g`

Reply:
Fixed in latest commits by @ximenesuk and @KoalaGeo.